count-note-beats.qml

```diff
@@ -380,6 +390,6 @@ MuseScore {
 		var doneMap = {};
 		applyToSelectionOrScore(labelBeat, measureMap, doneMap);
 
-		Qt.quit();
+		quit();
```

Comment:
Might use `return` here too, I believe

Reply:
I’ll have to look into `return` (though that won’t work everywhere); 3.2.3 needs `Qt.quit()` though.

I’ll probably split them all up into mu͒2/3/4 versions…

Reply:
Qt.quit() is wrong in any case even in 3.x, in 4.x is only leads to a crash

Reply:
Ah, I found an easier solution…